main.go

```diff
@@ -21,13 +23,63 @@ type Configs struct {
 	ResultPathPattern string `env:"result_path_pattern"`
 	Variant           string `env:"variant"`
 	Module            string `env:"module"`
+	Arguments         string `env:"arguments"`
+	CacheLevel        string `env:"cache_level,opt[none,only_deps,all]"`
 }
 
 func failf(f string, args ...interface{}) {
 	log.Errorf(f, args...)
 	os.Exit(1)
 }
 
+func getArtifacts(gradleProject gradle.Project, started time.Time, pattern string) (artifacts []gradle.Artifact, err error) {
+	for _, t := range []time.Time{started, time.Time{}} {
+		artifacts, err = gradleProject.FindDirs(t, pattern, true)
+		if err != nil {
+			return
+		}
+		if len(artifacts) == 0 {
+			if t == started {
+				log.Warnf("No artifacts found with pattern: %s that has modification time after: %s", pattern, t)
+				log.Warnf("Retrying without modtime check....")
+				fmt.Println()
+				continue
+			}
+			log.Warnf("No artifacts found with pattern: %s without modtime check", pattern)
+			log.Warnf("If you have changed default report export path in your gradle files then you might need to change ReportPathPattern accordingly.")
+		}
+	}
+	return
+}
+
+func exportArtifacts(deployDir string, artifacts []gradle.Artifact) error {
+	for _, artifact := range artifacts {
+		artifact.Name += ".zip"
```

Comment:
is it ok, to modify the artifact name? this ways its path does not follow the change.